### PR TITLE
fix(dgrid): Allow embedding of html by default

### LINKED ISSUE #3573: dgrid regression - embedded html
DGrid is "safeEncoding" embedded html by default, this should be optional.

### PR DESCRIPTION
**Pull request by GordonSmith**
Optionally disable (to ensure backward compatibility).

Fixes #3573

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [x] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: packages/dgrid/src/DBStore.ts [hunk 1]
```diff
@@ -12,6 +12,11 @@ export class DBStore {
         this._db = db;
     }
 
+    protected _renderHtml = false;
```
Comment (jbrundage) [inline packages/dgrid/src/DBStore.ts:15]: Shouldn't this be `true`?

File: packages/dgrid/src/RowFormatter.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ export class RowFormatter {
     private _columnIdx = {};
     private _formattedRow = {};
 
-    constructor(columns: ColumnType[]) {
+    constructor(columns: ColumnType[], protected _renderHtml) {
```
Comment (jbrundage) [inline packages/dgrid/src/RowFormatter.ts:49]: This new argument isn't used

Comment (GordonSmith) [reply packages/dgrid/src/RowFormatter.ts:49]: Yes it is, when you specify a "visibility" it implies that it is a Class Data Property (and it sets it)

File: packages/dgrid/src/RowFormatter.ts [hunk 3]
```diff
@@ -103,10 +103,10 @@ export class RowFormatter {
             }
         } else {
             if (this._formattedRow[column.field] === undefined) {
-                this._formattedRow[column.field] = "" + cell === undefined ? "" : safeEncode(cell);
+                this._formattedRow[column.field] = "" + cell === undefined ? "" : (this._renderHtml ? cell : safeEncode(cell));
```
Comment (GordonSmith) [inline packages/dgrid/src/RowFormatter.ts:106]: IOW its used here.

File: packages/dgrid/src/RowFormatter.ts [hunk 4]
```diff
@@ -103,10 +103,10 @@ export class RowFormatter {
...
             } else {
                 this._formattedRow[column.field] += LINE_SPLITTER;
-                this._formattedRow[column.field] += "" + cell === undefined ? "" : safeEncode(cell);
+                this._formattedRow[column.field] += "" + cell === undefined ? "" : (this._renderHtml ? cell : safeEncode(cell));
```
Comment (GordonSmith) [inline packages/dgrid/src/RowFormatter.ts:109]: ...and here

File: packages/dgrid/src/DBStore.ts (hunk 1 above)
Comment (GordonSmith) [reply packages/dgrid/src/DBStore.ts]: Yup (and fixed)

File: packages/dgrid/src/RowFormatter.ts (hunk 2 above)
Comment (jbrundage) [reply packages/dgrid/src/RowFormatter.ts:49]: Ahh okay

**Pull request merged**